include/path_tracking_pid/units.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <boost/units/systems/si.hpp>
+
+namespace path_tracking_pid::units
+{
+
+// Distance in meter.
+using distance_t = boost::units::quantity<boost::units::si::length, double>;
+
+// Distance squared in square meter.
+using distance_squared_t = boost::units::quantity<boost::units::si::area, double>;
+
+// Duration in second.
+using duration_t = boost::units::quantity<boost::units::si::time, double>;
+
+// Velocity in meter per second.
+using velocity_t = boost::units::quantity<boost::units::si::velocity, double>;
```

Comment:
What's the difference between `boost::units::quantity<boost::units::si::velocity, double>` and `boost::units::si::velocity`?

Reply:
There is no difference: double is the default value type. I just wanted to make it explicit we're using doubles.

Reply:
Ah, wait. May have misunderstood the question. Boost Units makes a distinction between units of measurement (like velocity, length, etc) and quantities, which are values of a certain unit of measurement.

- `boost::units::si::velocity` is a unit. and can't be used directly for storing values or calculations.
- `boost::units::quantity<boost::units::si::velocity, double>` is a quantity. a value of type `double` with the velocity unit of measurement.

`boost::units::quantity<boost::units::si::velocity, double>` is identical to `boost::units::quantity<boost::units::si::velocity>`.

For more info, see: https://www.boost.org/doc/libs/1_78_0/doc/html/boost_units/Quick_Start.html